Mathlib/NumberTheory/LSeries/HurwitzZetaEven.lean

```diff
@@ -189,6 +189,7 @@ lemma hasSum_int_cosKernel (a : ℝ) {t : ℝ} (ht : 0 < t) :
   simp only [this]
   exact hasSum_jacobiTheta₂_term _ (by rwa [I_mul_im, ofReal_re])
 
+open Classical in
```

Comment:
Instead of these `open Classical in`, I think you should register a `DecidableEq` instance for `UnitAddCircle`, once and for all (using classical itself).

Reply:
Good idea! Done.